src/styles/resultList.module.css

```diff
@@ -42,31 +42,30 @@
   border-bottom-right-radius: 5px;
 }
 
+.tabsContent::-webkit-scrollbar {
+  width: 0;
+  background: transparent;
```

Comment:
[-webkit-scrollbar is deprecated. Do not use it.](https://developer.mozilla.org/en-US/docs/Web/CSS/::-webkit-scrollbar) I'm not sure why you are using it at all.